ckanext/spatial/harvesters/base.py

```diff
@@ -230,7 +230,7 @@ def get_package_dict(self, context, data_dict):
 
         # Adding default_groups from config. This was previously not supported
         # by ckanext-spatial.
-        context = {'model': model, 'user': p.toolkit.c.user}
+        context = {'model': model, 'session': model.Session, 'user': self._get_user_name()}
```

Comment:
Is session required in Flask context now?

Reply:
I'm not sure. I'm following the pattern used by every other context setup in here and from the [CKAN docs](https://docs.ckan.org/en/2.9/extensions/plugins-toolkit.html#ckan.plugins.toolkit.ckan.plugins.toolkit.get_action). They actually suggest leaving it blank or passing in a null but I'm just going to follow the `ckanext-spacial` lead for now.